Scripts/Source/minai_Sex.psc

```diff
@@ -194,84 +194,97 @@ Function StartSexOrSwitchToGroup(actor[] actors, actor akSpeaker, string tags=""
   Main.Info("Sex: Starting/switching for " + actors.Length + " actors (tags: " + tags + ")")
   bool bSpeakerInScene = False
   bool bPlayerInScene = False
-  int i =0
-  int numMales = 0
-  int numFemales = 0
   bool bCanAnimate = True
-  Actor[] actorsInScene
-  sslThreadController Controller
   bool isNewScene = False
   if bHasOstim && minai_UseOStim.GetValue() == 1.0
+    Actor[] ostimActors = new actor[5]
```

Comment:
oStimActors is only 5 actors here, but down below you're creating an array with potentially many more entries in it (In the orgy command). How many does ostim support at once?

Reply:
You should use the same size in both places or you'll encounter errors later. Doesn't matter to me which size.

Reply:
The orgy command can go away as this new function will handle an array of 5.